airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -368,6 +371,11 @@ def get_namespace(self) -> str | None:
             return namespace
         return None
 
+    def get_xcom_sidecar_container_image(self):
+        """Returns the xcom sidecar image that defined in the connection"""
+        image = self._get_field("xcom_sidecar_container_image")
+        return image if image else "alpine"
```

Comment:
After making image optional in add_xcom_sidecar, i would do this instead
```suggestion
    def get_xcom_sidecar_container_image(self):
        """Returns the xcom sidecar image if defined in the connection"""
        return self._get_field("xcom_sidecar_container_image") or None
```

Reply:
No need to append `or None`, as the `_get_field` already returns `None` for missing values.